great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -378,10 +395,73 @@ def _check_renderer_methods(
     @staticmethod
     def _check_linting(expectation_instance) -> ExpectationDiagnosticCheckMessage:
         """Check if linting checks pass for Expectation"""
-        # TODO: Perform linting checks instead of just giving thumbs down
+        sub_messages = []
+        passed = False
+        black_ok = False
+        isort_ok = False
+        file_and_class_names_ok = False
+
```

Comment:
Could we add type annotations to all variable declarations throughout?
```suggestion
        sub_messages: List[dict] = []
        passed: bool = False
        black_ok: bool = False
        isort_ok: bool = False
        file_and_class_names_ok: bool = False
```